### ISSUE TITLE
process: process.exit should immediately exit

### ISSUE THREAD

**Issue by yorkie** [yodaos-project/ShadowNode#373]
* **Version**: 0.10.x
* **Platform**: all
* **Subsystem**: process

As Node.js Documentation shows the correct behavior:

> Calling `process.exit()` will force the process to exit as quickly as possible even if there are still asynchronous operations pending that have not yet completed fully, including I/O operations to process.stdout and process.stderr.

Referenced at #372.

**Comment by legendecas** [comment]
Can not re-produce in latest version of ShadowNode, closing for now.

**Comment by legendecas** [comment]
Latest v0.11.x could reproduce with follow snippet:

```js
process.exit(233)
console.log('foo') // expected unreachable code. yet it's executed.
```

**Comment by lolBig** [comment]
fixed by #554
